loops-and-arrays.js

```diff
@@ -42,6 +65,17 @@ const numbers = [2, 4, 5, 29, 38];
 
 const squares = [30, 2, 8, 24, 11];
 
+console.log("Voor het script opdracht 3:");
+console.log(squares);
+
+for (let i = 0; i < squares.length; i++) {
+    squares[i] = "Het volume van " + squares[i] + " is " + (squares[i] ** 3);
+}
+
+console.log("Na het script:");
+console.log(squares);
+
```

Comment:
ziet er keurig uit. Novi voegde op regel 72 "const volume =" toe, maar jouw code logt hetzelfde resultaat, dus in mijn optiek is het beide goed.